src/main/java/ru/octol1ttle/flightassistant/indicators/AltitudeIndicator.java

```diff
@@ -54,7 +54,12 @@ public void render(DrawContext context, TextRenderer textRenderer) {
 
         if (FAConfig.indicator().showGroundAltitude) {
             Color color = data.altitude() < safeLevel ? FAConfig.indicator().warningColor : FAConfig.indicator().frameColor;
-            drawText(textRenderer, context, Text.translatable(data.groundLevel == data.voidLevel() ? "flightassistant.void_level" : "flightassistant.ground_level"), xAltText - 10, bottom, color);
+            if (!data.isCurrentChunkLoaded) {
+                drawText(textRenderer, context, Text.literal("?"), xAltText - 10, bottom, FAConfig.indicator().cautionColor);
+            }
+            else {
+                drawText(textRenderer, context, Text.translatable(data.groundLevel == data.voidLevel() ? "flightassistant.void_level" : "flightassistant.ground_level"), xAltText - 10, bottom, color);
+            }
```

Comment:
This should show a placeholder instead. similar to those displayed when an indicator failure occurs (see `renderFaulted`)